backup/moodle2/backup_quizaccess_seb_subplugin.class.php

```diff
@@ -75,6 +82,9 @@ protected function define_quiz_subplugin_structure() {
             $contextid
         );
 
+        $params = ['id' => '../templateid'];
+        $subplugintemplatesettings->set_source_table('quizaccess_seb_template', $params);
```

Comment:
should we do here the same way as we do with quiz_settings ? like quizaccess_seb\template::TABLE?